src/mobu/storage/process.py

```diff
@@ -0,0 +1,57 @@
+"""Class with method to execute noninteractive subprocesses asynchronously."""
+
+import asyncio
+from pathlib import Path
+from shlex import join
+
+from structlog.stdlib import BoundLogger
+
+from ..exceptions import AsyncioProcessError
+from ..models.user import AuthenticatedUser
+
+
+class Process:
+    """A thin wrapper around asyncio.subprocess.create_process_exec."""
+
+    def __init__(
+        self,
+        *,
+        user: AuthenticatedUser | None = None,
+        logger: BoundLogger | None = None,
+    ) -> None:
+        self._user = user
+        self._logger = logger
+
+    async def exec(
+        self,
+        cmd: str,
+        *args: str,
+        cwd: Path | None = None,
+    ) -> asyncio.subprocess.Process:
```

Comment:
The return value here is not currently being used, so I would start with the simplest interface that will work, don't return a value, and we can always make it more complicated later if we have to.